scripts/publish.sh

```diff
@@ -23,6 +27,18 @@ else
     DEPLOY_ENVIRONMENT=${branch_environment_map[$BRANCH]}
 fi
 
+if [[ $BRANCH == "master" ]]; then
+  if ! ci-append-sha --include-count; then
+    echo "ci-append-sha failed! Exiting..."
+    exit $FAILED_SETUP
+  fi
+else
+  if ! ci-append-sha --include-count --tag ${BRANCH}; then
```

Comment:
adding `--tag` should make the artifacts a little bit easier to differentiate between prod artifacts and staging ones.